client/session.go

```diff
@@ -1177,6 +1177,81 @@ func (s *session) FetchBootstrapBlocksFromPeers(
 	return result.result, nil
 }
 
+func (s *session) FetchRepairBlocksFromPeers(
+	namespace ts.ID,
+	shard uint32,
+	repairBlocks []block.ReplicaMetadata,
```

Comment:
lol, so turns out `blocksMetadata` is defined as a struct in this package. Changing var name to `requiredBlocksMetadata`. Lemme know if you have a better suggestion.